Lib/pydoc.py

```diff
@@ -2074,19 +2074,21 @@ def help(self, request, is_cli=False):
 
     def intro(self):
         self.output.write('''
-Welcome to Python {0}'s help utility!
-
-If this is your first time using Python, you should definitely check out
-the tutorial on the internet at https://docs.python.org/{0}/tutorial/.
+Welcome to Python {0}'s help utility! If this is your first time using
+Python, you should definitely check out the tutorial on the internet at
+https://docs.python.org/{0}/tutorial/.
 
 Enter the name of any module, keyword, or topic to get help on writing
-Python programs and using Python modules.  To quit this help utility and
-return to the interpreter, just type "quit".
+Python programs and using Python modules.  To get a list of available
+modules, keywords, symbols, or topics, type "modules", "keywords",
+"symbols", or "topics".
+
+Each module also comes with a one-line summary of what it does; to list
+the modules whose name or summary contain a given string such as "spam",
+type "modules spam".
 
-To get a list of available modules, keywords, symbols, or topics, type
-"modules", "keywords", "symbols", or "topics".  Each module also comes
-with a one-line summary of what it does; to list the modules whose name
-or summary contain a given string such as "spam", type "modules spam".
+To quit this help utility and return to the interpreter,
+type "q" or "quit".
```

Comment:
I think we should replace "type" with "enter" in every case here.  The latter helps to clarify that you must hit the 'enter' key after your input; the former implies that is not necessary.

Reply:
Agree.  3 replacements